static/src/javascripts/projects/common/modules/commercial/adverts.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'bean',
+    'qwery',
+    'fastdom'
+], function (bean, qwery, fastdom) {
+    var shouldWePolyfill = !('open' in document.createElement('details'));
+
+    return {
+        init: init
+    };
+
+    function init() {
+        if (shouldWePolyfill) {
```

Comment:
oh I see. Not in this case, it is native behaviour that we want to simulate, so it is bound to all `SUMMARY` elements.